pkg/api/v20170701/types.go

```diff
@@ -5,6 +5,20 @@ import (
 	"strings"
 )
 
+// Add the validate tag to add a mark, if this is a required or optional field
+// Current it is not used in any logic yet
+// TODO, we could possibly use this tag in all the Validate functions
+const (
+	ValidateTag = "validate"
```

Comment:
I will use whatever the validator package specified. My other comment has the options. Pick one that you favor more. They have different syntax.